src/renderer/extensions/vueNodes/components/LGraphNode.vue

```diff
@@ -101,7 +101,11 @@
       >
         <!-- Slots only rendered at full detail -->
         <NodeSlots
-          v-memo="[nodeData.inputs?.length, nodeData.outputs?.length]"
+          v-memo="[
+            nodeData.inputs?.length,
+            nodeData.outputs?.length,
+            executionStore.lastNodeErrors
+          ]"
```

Comment:
It's pretty rare that you actually need v-memo. Could you see if it's necessary?

Reply:
i actually removed it during debugging but put it back because i figured i'd get chastised for removing it :Joy:

Reply:
I think we can remove it, happy to follow on

Reply:
no remove then.